apps/versions/models.py

```diff
@@ -490,6 +490,10 @@ def sign_files(self):
         if self.addon.type == amo.ADDON_EXTENSION:
             for file_obj in self.files.all():
                 packaged.sign_file(file_obj)
+        # They can have multi-package XPIs.
+        elif self.addon.type == amo.ADDON_THEME:
+            for file_obj in self.files.filter(is_multi_package=True):
+                packaged.sign_file(file_obj)
```

Comment:
From what @kmaglione and I have seen, you can only have multi-package XPIs detected as either complete themes or extensions (the code is in https://github.com/mozilla/olympia/blob/75c8700e71de14848bc2d2afbf61362c6c7497c9/apps/files/utils.py#L174).